File: Python/DSA Sheet/Arrays/ReverseArray.py
def reverse(arr):
    low = 0
    high = len(arr) - 1
    while low < high:
        arr[low], arr[high] = arr[high], arr[low]
        low += 1
        high -= 1
        
    return arr

if __name__ == "__main__":
    arr = [1, 2, 3, 4, 5]
    print(reverse(arr))  # Output: [5, 4, 3, 2, 1]

File: JAVA/DSA/reverse_Array.java
package DSA;

public class reverse_Array {

    public static int reverse(int arr[], int n){
        int low = 0;
        int high = n;

        while(low <= high){
            for(int high = arr.length; high >= 0; high--){
                int temp = arr[low];
                int arr[low] = arr[high];
                int arr[high] = temp;
            }
        }
    }

    public static void main(String args[]){
        int arr[] = {1,4,3,2,6,5};
        int n = arr.length;
        
    }
}
